model/src/main/java/org/perfrepo/dto/test_execution/TestExecutionSearchParams.java

```diff
@@ -1,10 +1,73 @@
 package org.perfrepo.dto.test_execution;
 
+import org.perfrepo.model.to.OrderBy;
+
+import java.util.Set;
+
 /**
  * Data transfer object that contains filter parameters for {@link TestExecutionDto} searching query.
  *
  * @author Jiri Grunwald (grunwjir@gmail.com)
  */
 public class TestExecutionSearchParams {
```

Comment:
I think TestExecutionSearchCriteria is more descriptible, but that's a minor comment.